root/components/forms.tt

```diff
@@ -352,7 +352,12 @@
       [% FOREACH id IN form.options %]
          [%- id = id.value -%]
          <span [% IF form.value != id %] style="display:none" [% END %] class="type-bubble-description" id="type-bubble-description-[% id %]">
-           <strong>[%- add_colon(l('Description')) -%] </strong>[%- types.$id.l_description -%]
+           <strong>[%- add_colon(l('Description')) -%] </strong>
+           [% IF types.$id.l_description %]
+             [%- types.$id.l_description -%]
+           [% ELSE %]
+             [%- lp('(none)', 'description') -%]
```

Comment:
Or “Style BDFL has been lazy”? Or just nothing?

Reply:
Nothing would show an empty bubble. Right now, only "Other" (which arguably shouldn't exist) shows this, IIRC.

Reply:
Reported in MBS-11177.